### PR TITLE
add error query params to SlackActionFail redirects

### PR DESCRIPTION
**Pull request by camhashemi**
@leogrim @ryanpbrewster @andrewconner 

one way of addressing issues here https://app.asana.com/0/32728208409723/86741480256014

this was quick, now the site can parse the param and add a global banner with error messages. but I don't want to spend substantial time on it if we want to go with specific error page redirects. so some feedback would be nice.

### CODE REVIEW

**Comment by andrewconner** [comment]
Works for me.

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/SlackOAuthController.scala [hunk 1]
```diff
@@ -66,10 +71,10 @@ class SlackOAuthController @Inject() (
     }.recover {
       case fail: SlackActionFail =>
         log.warn(s"[SlackOAuthController#handleAsBrowserRequest] Error: ${fail.code}")
-        Redirect(fail.bestEffortRedirect.fold("/")(_.absolute))
+        Redirect(fail.bestEffortRedirect.fold(s"/?error=${fail.code}")(_.absolute))
       case badFail =>
         airbrake.notify("Uncategorized failure for SlackAction", badFail)
-        Redirect("/")
+        Redirect("/?error=unknown")
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/SlackOAuthController.scala:77]: Do you want the `/?` or just `?`

Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/SlackOAuthController.scala:77]: Looks like they're equivalent. Learn something new every day

Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/SlackOAuthController.scala:77]: you ask good questions

Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/SlackOAuthController.scala:77]: Yeah it's the same. A benefit of `/?...` is that the destination isn't relative.

**Pull request merged**